script/src/verify.rs

```diff
@@ -591,23 +597,23 @@ mod tests {
 
         // Not enough cycles
         assert_error_eq(
-            verifier.verify(100).unwrap_err(),
+            verifier.verify(11).unwrap_err(),
```

Comment:
FYI, we should define these cycles as meaningful constant variables later.

Reply:
Updated, please review again, thanks.